types/tizen-tv-webapis/lib/billing.d.ts

```diff
@@ -0,0 +1,390 @@
+ */
+interface BillingVerifyInvoiceCallback {
+    (data: VerifyInvoiceData): void;
+}
+
+/**
+ * Callback method returning availability of country list status.
+ * @param data It includes getServiceCountryAvailability API result. but you can't use it as it is. you have to parse it as JSON format data.
+ */
+interface BillingGetServiceCountryAvailabilityCallback {
+    (data: ServiceCountryAvailabilityData): void;
+}
+
+/**
+ * Callback method returning the purchase history request status.
+ * @param data It includes getUserPurchaseList API result. but you can't use it as it is. you have to parse it as JSON format data.
+ */
+interface BillingGetUserPurchaseListCallback {
+    (data: UserPurchaseData): void;
+}
+
+/**
+ * Callback method returning the subscription cancel request status.
+ * @param data It includes cancelSubscription API result. but you can't use it as it is. you have to parse it as JSON format data.
+ */
+interface BillingCancelSubscriptionCallback {
+    (data: CancelSubscriptionData): void;
+}
+
+/**
+ * Callback method returning the service availability check status.
+ * @param data It includes isServiceAvailable API result. but you can't use it as it is. you have to parse it as JSON format data.
+ * @note `deprecated` 5.5
+ */
+interface BillingIsServiceAvailableCallback {
+    (data: ServiceAvailableData): void;
+}
+
+/**
+ * This module defines the Billing (Samsung Checkout) functionalities provided by the Tizen Samsung TV Product API.
+ * @privilegeLevel Public
+ * @privilegeName http://developer.samsung.com/privilege/billing
+ * @since 2.4
+ */
+// tslint:disable-next-line strict-export-declare-modifiers
+declare interface BillingManager {
+    /**
+     * Retrieves the Billing API version.
+     * @returns Billing API version
+     * @privilegeLevel Public
+     * @privilegeName http://developer.samsung.com/privilege/billing
+     * @throw WebAPIException SecurityError, UnknownError
+     * @since 4.0
+     */
+    getVersion: () => string;
+
+    /**
+     * Enables implementing the Samsung Checkout Client module within the application. After authenticating the purchase information through the application, the user can proceed to purchase payment.
+     * @param appId Application ID
+     * @param serverType Billing server type
+     * @param paymentDetails Payment parameters
+     * OrderItemID[MANDATORY]/OrderTitle[MANDATORY]/OrderTotal[MANDATORY]/OrderCurrencyID[MANDATORY]/OrderID[OPTIONAL]/OrderCustomID[MANDATORY]
+     * @param onsuccess Returns "payResult" and "payDetail" if there is no internal error occurs until client to server data communication.
+     * payResult, can still contains error when billing server confirms that the given parameters does not have expected value or have problem while processing it
+     * payDetail, can have additional data when it's returned, such as InvoiceID. Please refer to the development guide of "buyItem"	for details
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/tizen-tv-webapis/lib/billing.d.ts;
  Line: 185;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.